tobago-core/src/main/java/org/apache/myfaces/tobago/internal/component/AbstractUISheet.java

```diff
@@ -84,24 +84,17 @@ public abstract class AbstractUISheet extends AbstractUIData
   private transient MeasureList columnLayout;
   private transient boolean autoLayout;
   private transient boolean lazyUpdate;
+  private transient int lazyFirstRow;
 
   private transient Grid headerGrid;
 
   @Override
   public void encodeAll(FacesContext facesContext) throws IOException {
 
     if (isLazy()) {
-      if (getRows() == 0) {
-        LOG.warn("Sheet id={} has lazy=true set, but not set the rows attribute!", getClientId(facesContext));
-      }
-      if (getShowRowRange() != ShowPosition.none) {
```

Comment:
Do we really want to delete the other warnings?

Reply:
We can keep the warnings for "lazy=true AND rows > 0". I will change this.